openslides_backend/permissions/management_levels.py

```diff
@@ -0,0 +1,47 @@
+from enum import Enum
+
+from ..shared.exceptions import PermissionException
+
+
+class CompareRightLevel(str, Enum):
+    def __new__(cls, value: str, weight: int):  # type: ignore
+        obj = str.__new__(cls, value)  # type: ignore
+        obj._value_ = value
+        obj.weight = weight
+        return obj
+
+    def check_instance(self, other: str) -> None:
+        if not isinstance(other, self.__class__):
+            raise PermissionException(
+                f"The comparison expect an {self.__class__}-type and no string!"
+            )
+
+    def __lt__(self, other: str) -> bool:
```

Comment:
Are all operators in actual usage? I can see that `>=` is needed since it is the basic check ("has the user the right OML?"). Especially `>` from small typos can lead to unwanted errors. If the other operators are not used I would like to raise an "NotImplementedError" in the operator implementations. **If** someone needs them for whatever reason, it can be implemented simply and until than, mistakes are hopefully not so common.

Reply:
I disagree: They are already implemented now and it is convenient if someone needs them. And sorry, but if someone mistypes `>` instead of `>=`, I strongly hope that either the tests or the reviews will catch this. The same issue is for all numbers also, btw; would you argue to overwrite the `>=` operator on `int` to raise a `NotImplementedError` if you don't need it in your program? ;)

Reply:
We are the beginning with the OML and CML-permissions checking, the standard for CML is still not written. Therefore I agree with @jsangmeister